### ISSUE TITLE
SCRFD evalution in origin size(WIDERFACE)?

### ISSUE THREAD

**Issue by Wwupup** [deepinsight/insightface#1938]
Hello,
https://github.com/deepinsight/insightface/blob/478aafb4fc66030e07fb46143e8e069f85e68147/detection/scrfd/tools/scrfd.py#L26
It seems that the postprocess limits input-size in times of 64. I meet error when detecting img in size (640, 481). So how can I evalute widerface in origin size(README has shown the results of origin size). In addition, I test wideface validation with this code(https://github.com/deepinsight/insightface/blob/478aafb4fc66030e07fb46143e8e069f85e68147/detection/scrfd/tools/scrfd.py) in VGA(640, 480) resolution but get low score than noted in README. I wonder if any TTA(such as flip) do you use in testing process.
Looking forward to your answer. Thanks.

**Comment by nttstar** [comment]
For testing with the original size, we should always make the input size divisible by 32 or 64. For testing with VGA resolution, we simply resize the longer side to 640 pixels.

**Comment by WassimBouatay** [comment]
@Wwupup I am getting 0.711 mAP on the validation set on WIDER. I am using resolution 640x640, i even tried 1024x1024 but i only got 0.8x. Can you tell me how did you reproduce the performances ?  

I used the pretrained model for scrfd-34g in here:  https://github.com/deepinsight/insightface/tree/master/detection/scrfd

**Comment by Wwupup** [comment]
> @Wwupup I am getting 0.711 mAP on the validation set on WIDER. I am using resolution 640x640, i even tried 1024x1024 but i only got 0.8x. Can you tell me how did you reproduce the performances ?
> 
> I used the pretrained model for scrfd-34g in here: https://github.com/deepinsight/insightface/tree/master/detection/scrfd

I just use the official pre-training weights directly, without any other modifications. You should make sure `score_thresh = 0.02` or other value close to 0, because the impact of this threshold is significant.

**Comment by WassimBouatay** [comment]
I just found out why. I used test.py as I am used to in mmdetection and somehow I get 0.71. I get the right values when I use test_widerface.py
